production/exit-node-1/sealed-secret.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: exit-node-1
+  namespace: production
+spec:
+  encryptedData:
+    HOPRD_API_TOKEN: AgA2escn339OiPlV2S8vHEHw5qjMbG7PA9/hT/eZMDa46lkCqXXQaijDlATMqfX3r9GgAX4l9iDbp6TdCiDS2z08n+iGXX6xRxLeGfpMgkLcoUL9ZaqaKlPQqUj+zH4xJoy9OIRjnmpnKe1LY6FBnwDGnxnoP3J0JqZwXtr8lJB4yKf+s6NKp0/e4UD/T+D0xdn9pI1RHS9GabXsg9CpHWENiHXyM0PKElR+cLOCliOrqbbvqtAE6weh/EvYQB0+ElbyX/qq+g/a4i2mDrCbsHRqJTap/VMaGHswh9uI6H1NE8jqf2vEo4zMnTfW+KyxRQ+aH0uOhSBaRTOgYcEQKU9PARjJMmD2PUyziV5Z9oJIo7FGZvDDT9W2o/C/IR6LtVHFBvQnQumwV250Q4Y0RbSpNOxvsFVBSbgbORpMnY6RmsX75yJXNvb4scsCMHgpoYC8lHIkPSIyYks8oyo3aombRilH25fVpqFxY5FXMVyHgWjxPqpzvxvqqeXTuXbcLCxr8zupPJrd74fPBJVmo+5r5JZtdas6cUuI9UxyzDs+hDiQYvBK8eW5MihTQzwsoSUFVN52h2Dl1PV+Qx/O/tCTnxttVxS0sQF3ZQNbewy3n7dzi5CAexS2Ro2QZ9bxzcpTXZK85kZNAUes4UzdH+OuadDm4SJOCro6WQKf2ifyFvfgJZ1vgOddmWCepSTFnQEJTEbTyA==
+    HOPRD_PASSWORD: AgArQ/0OJsjJFKy3p4r2TJ5N1Gyie0Yav2OHo5s2ejW2xaX7TQXq/vArbgSB1lEuhvT3ZUTl/gWRfiA+MjsggLBqTuPPMjmnimjc6mWegOZVlO7U37QQ7oIDA1BZ5q6SF2jqVkluS6WM54PlG+ELqyxrpe05W6Jb3MgGxllUgCJUgv5hpMT3UtjiAt5gCc/R1nQi6y4l8qPwa6qACPH630GGlfLwEO2NOxLdjnW5PR+ljWylhRZTALw7z3IQgZIcmNTHHxP4mWs0Zsbq6dUsorFJthhcmfcm//cKd2sgJH7rXiz55C0Kbj2jn6JvWUlxLLDTTPi5Pc8qX4gIrVD99C3HlqJbFL3/Pq9d+FY6cbcdgyz4SH3rg6KLylmPR+wHO+n4wQezUqb4dgeoguHM2XtLNQNgfMVyJ6jsYY5wY9Wd4xaRT7PdA93PQx/uJqRN++q3PsC0/hjNZOYFZg5UAIb6BgS+UuCWIQ7ylsJ5keK2U6bpSZf/O0or3KuMhk4OT0HRKYj9AWMjQmQzSFTfi86ucYMmYb/tV4BZSBnGepOz2JBwivPTZc95EVkQvgM+lGnVBr8nhRW/Jw7A554HUAbqsRfqqCAhogWdlOKsUm8Aff28uDD5+kHxed2qHTQw9DGgBIplF6clcZY4vM3r7dmkBvH0RzzVj68fGmiUV9JgGyIuv3lOUtbsCQZFSaA5NTaTsU+15w==
```

Comment:
Why do you need to store the password of Hoprd identity file?
I thought that you only interacted with the API with hoprd nodes?

Reply:
We should not be using a custom private key for HOPRd.
I think this was added by mistake because we need to do that for the RPCh exit node.

RPCh exit node has its own identity which similarly to HOPRd requires a password to encrypt it.
Unlike HOPRd, it does _not_ have an API, making it hard to retrieve the `public key` of the RPCh node.

We have two options available to us:

1. pre-create an identity and pass it manually (this way we know the public key already)
2. scrape the `public key` from the exit-node logs (something similar to [this](https://github.com/Rpc-h/scripts/blob/591a714746836ca6da3f31257ba4de5b98feada4/common.sh#L70))